app/modules/transactions.js

```diff
@@ -25,15 +41,22 @@ export const setIsLoadingTransaction = (isLoading: boolean) => ({
   }
 })
 
-export const syncTransactionHistory = (net: NetworkType, address: string) => async (dispatch: DispatchType) => {
+export const syncTransactionHistory = (
+  net: NetworkType,
+  address: string
+) => async (dispatch: DispatchType) => {
   dispatch(setIsLoadingTransaction(true))
-  const [err, transactions] = await asyncWrap(api.neonDB.getTransactionHistory(net, address))
+  const [err, transactions] = await asyncWrap(
+    api.neonDB.getTransactionHistory(net, address)
+  )
   if (!err && transactions) {
-    const txs = transactions.map(({ NEO, GAS, txid, block_index }: TransactionHistoryType) => ({
-      txid,
-      [ASSETS.NEO]: NEO,
-      [ASSETS.GAS]: GAS
-    }))
+    const txs = transactions.map(
+      ({ NEO, GAS, txid, block_index }: TransactionHistoryType) => ({
+        txid,
+        [ASSETS.NEO]: NEO,
+        [ASSETS.GAS]: GAS
```

Comment:
The call to `api.neonDB.getTransactionHistory` returns numbers for `NEO` (int) and `GAS` (float).  I think we should convert these similar to what you're doing elsewhere:

```js
  [ASSETS.NEO]: toFixedDecimals(NEO, 0)
  [ASSETS.GAS]: toBigNumber(resultBalance.GAS.balance).round(COIN_DECIMAL_LENGTH).toString()
```

That has some implications on the caller of `syncTransactionHistory` that we would have to update as well.